Branch-TestBed/ANSCompatibility.h

```diff
@@ -0,0 +1,31 @@
+//
```

Comment:
do we need these files in the testbed? or were you just testing if there'd be a conflict?